base/Plain_netCDF_Time.F90

```diff
@@ -1,20 +1,27 @@
+!-------------------------------------------------------------------
+! Note: use for OSSE project
+!   File to be replaced by more systematic implementations.
+!   It contains codes for time conversion and bisect.
+!-------------------------------------------------------------------
+
 #include "MAPL_Exceptions.h"
 #include "MAPL_ErrLog.h"
 #include "unused_dummy.H"
 
 !
 !>
-!### MODULE: `MAPL_plain_netCDF_Time`
+!### MODULE: `Plain_netCDF_Time`
 !
 ! Author: GMAO SI-Team
 !
-module MAPL_plain_netCDF_Time
+module Plain_netCDF_Time
```

Comment:
Is there a reason to not keep the conventional `mapl` suffixe for this module?

Reply:
I was thinking that this file is a temporary file holder for plain netCDF files that will be replaced by PFIO metadata in terms of group treatment. It also contains some special treatment for time units conversion and time utilities functions that @darianboggs had been working on.   I think this file is a convenient tool and will need to be consolidated to other modules in the future. Also it is not intensively using ESMF, so I think it does not qualify the MAPL term.

Reply:
OK - if it is temporary don't fix for now.

Note the MAPL prefix has nothing to do with "using ESMF".   It is entirely about managing name space.  (Well, that and observing conventions in style.)